packages/sdk/src/conversation/command.ts

```diff
@@ -22,7 +30,22 @@ export class CommandBot {
    * @param options - initialize options
    */
   constructor(adapter: BotFrameworkAdapter, options?: CommandOptions) {
-    this.middleware = new CommandResponseMiddleware(options?.commands);
+    let ssoCommandActivityHandler: SsoExecutionActivityHandler | undefined;
+    if (options?.ssoConfig?.CustomSsoExecutionActivityHandler) {
+      ssoCommandActivityHandler = new options.ssoConfig.CustomSsoExecutionActivityHandler(
+        options?.ssoConfig
+      );
+    } else if (options?.ssoCommands?.length && options?.ssoCommands?.length > 0) {
```

Comment:
```suggestion
    } else if (options?.ssoCommands?.length > 0) {
```
Can it be simplified to this one?

Reply:
Cannot simplified as this, because it will show error as below in typescript:
![image](https://user-images.githubusercontent.com/5545529/187162352-d2db6ecf-566f-4b3f-97aa-b65ed3fa88a0.png)